pkg/utils/gitops/controller.go

```diff
@@ -4,12 +4,14 @@ import (
 	"context"
 	"crypto/tls"
 	"fmt"
-	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
 	"net/http"
 	"time"
 
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+
 	routev1 "github.com/openshift/api/route/v1"
 	kubeCl "github.com/redhat-appstudio/e2e-tests/pkg/apis/kubernetes"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
```

Comment:
this has moved to https://github.com/redhat-appstudio/application-api/tree/main/api/v1alpha1

Reply:
I've replaced the import with ur suggested link and it just fail on the go get package

Reply:
That is the link to see the repo in your browser. For the import, try `github.com/redhat-appstudio/application-api/api/v1alpha1`